### ISSUE TITLE
Add example exercise answers to wiki?

### ISSUE THREAD

**Issue by skokenes** [calmm-js/partial.lenses#213]
Hi, I'm working through the exercises at https://calmm-js.github.io/partial.lenses/exercises.html and am finding them really helpful. However, as I get towards the end I am finding that I'm not sure about what I'm doing exactly, even if I can get the correct answer. For example, I struggled for a while with figuring out the isomorphism example. I ended up with an answer like this that passed the test:

```javascript
const a = L.conjugate(L.json(),L.pick({bar: ["foo", L.array(L.complement)]}));
const b = L.conjugate(L.json(), L.pick({foo: ["bar", L.array(L.complement)]}));
const iso = L.iso(L.get(a), L.get(b));
```

My issue now is that I have no clue of the "quality" of the answer. Have you thought at all about putting example solutions to these exercises in the wiki? It would be a nice resource for beginners like myself to learn from.

**Comment by polytypic** [comment]
> I struggled for a while with figuring out the isomorphism example.

Yes, it is probably one of the trickiest exercises on the page.  There is a simpler answer...

> Have you thought at all about putting example solutions to these exercises in the wiki?

That is a good idea.  Could you create the page in the wiki with with your sample answers?  I'll review the page and provide additional sample answers.

**Comment by skokenes** [comment]
Ok here are my novice solutions to the problems https://github.com/calmm-js/partial.lenses/wiki/Exercise-Answers

Looking forward to seeing the alternatives

**Comment by polytypic** [comment]
Thanks!  I edited some of the answers and made a couple of additions.  There really should be more exercises...  Suggestions for exercises are welcome!

**Comment by skokenes** [comment]
I don't understand lenses well enough yet to come up with the exercises but if someone can produce rough ideas for them, I'm happy to clean them up with sample code, tests, prompts and hints. Just really need someone to give me the seed idea, like "Use a lens to do x,y,z"
